features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/blocks/subscription/subscription-list/ajax/subscription-list.jag

```diff
@@ -144,7 +144,58 @@ include("/jagg/jagg.jag");
             }
         }
         print(obj);
-     }else {
+    }else if (action == "getSubscriptionByAPI"){
+
+        var appname = request.getParameter("app");
+
+        if (!user) {
+            if(!ssoEnabled){
+        		print({
+        		    error:true,
+        		    message:msg.error.loginRequired(action)
+                    });
+        	}
+        	return;
+        }
+        username = user.username;
+        var log = new Log();
+        log.info(username);
```

Comment:
may be this should be a debug log or we can remove this if this is for testing purposes. 
It's best to initialize the log with the name to identify where it cause the issue like var log = new Log('subscriptions list');